server/routes/stats_router.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ResponseError } from '@opensearch-project/opensearch/lib/errors';
+import {
+  IOpenSearchDashboardsResponse,
+  IRouter,
+  SavedObjectsFindResponse,
+} from '../../../../src/core/server';
+import { APP_API, APP_PATH, MAP_SAVED_OBJECT_TYPE } from '../../common';
+import { getStats } from '../common/stats/stats_helper';
+import { MapSavedObjectAttributes } from '../../common/map_saved_object_attributes';
+
+export function statsRoute(router: IRouter) {
+  router.get(
+    {
+      path: `${APP_API}${APP_PATH.STATS}`,
+      validate: false,
```

Comment:
why we are making this as false?

Reply:
Because now the stats endpoint don’t require params/query/body pass in, so we don't need it